groundzero/src/constants/index.js

```diff
@@ -3,7 +3,17 @@ This file contains list of static texts that will be displayed in the web page.
 In the future, please use this file as the only way to interact with editing texts.
 */
 
-import { groundzero, facebook, instagram, linkedin, twitter, airbnb, binance, coinbase, dropbox } from "../assets";
+import airbnb from "../assets/airbnb.png"
+import binance from "../assets/binance.png"
+import coinbase from "../assets/coinbase.png"
+import dropbox from "../assets/dropbox.png"
+import facebook from "../assets/facebook.svg"
+import groundzero from "../assets/groundzero.png"
+import instagram from "../assets/instagram.svg"
+import linkedin from "../assets/linkedin.svg"
+import twitter from "../assets/twitter.svg"
```

Comment:
I also tried fixing this import issue in the latest PR, I think it's neater to maintain a index.jsx file for assets so that we don't have long import statements in external files

Reply:
Sure, thanks for the suggestion. I have fixed upon pulling from your merged PR :D